server/httpsrv/httpsrv.go

```diff
@@ -169,6 +170,7 @@ func (s *T) handleProduce(w http.ResponseWriter, r *http.Request) {
 	// Get the message body from the HTTP request.
 	var msg sarama.Encoder
 	if msg, err = s.readMsg(r); err != nil {
+		log.Infof("Rejecting HTTP message body for produce request from %s: err=(%s)", r.RemoteAddr, err)
```

Comment:
Please use the benefits of structural logging with sirupsen/logrus by logging errors as such: `log.WithError(err).Infof(...`.